sunpy/map/mapbase.py

```diff
@@ -1752,36 +1758,36 @@ def draw_rectangle(self, bottom_left, width: u.deg, height: u.deg, axes=None, **
 
         Notes
         -----
-
         Extra keyword arguments to this function are passed through to the
         `~matplotlib.patches.Rectangle` instance.
-
         """
+        if any(x is None for x in (width, height)):
```

Comment:
```suggestion
        if width is None or height is None:
```
Unless there's some subtlety that I'm missing, I think that simply writing it out is both more readable and shorter.

Reply:
Earlier I was checking top_right here as well.